src/main/java/core/basesyntax/dao/impl/FruitStorageDaoImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorageDaoImpl implements FruitStorageDao {
+    @Override
+    public void addFruit(FruitTransaction fruitTransaction) {
```

Comment:
The `addFruit` method is replacing the quantity of fruit in the storage. This might not be the expected behavior if you want to add more fruits to the existing stock. Consider if you need to sum the quantities instead.